CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ Features:
 
 Bug Fixes:
 - When using CMake presets, the Project Status View now shows the build target along with the build preset. [PR #3241](https://github.com/microsoft/vscode-cmake-tools/pull/3241)
+- IntelliSense resolves headers coming from MacOS frameworks. CMake 3.27 or later is required. [#2324](https://github.com/microsoft/vscode-cmake-tools/issues/2324)
 
 Improvements:
```

Comment:
Not for this PR, but before we release 1.15 I think we should review this list to be more consistent about what is a "bug fix" and what is an "improvement".

Reply:
I would agree that we should be more consistent and make sure we all agree on it. For example, this strikes me as an "improvement" 😂